simavr/sim/sim_vcd_file.c

```diff
@@ -311,11 +310,12 @@ avr_vcd_init_input(
 					AVR_LOG(vcd->avr, LOG_WARNING,
 							"%s IRQ was not found\n",
 							vcd->signal[i].name);
-				continue;
+			} else {
+				AVR_LOG(vcd->avr, LOG_WARNING,
+						"%s is an invalid IRQ format\n",
+						vcd->signal[i].name);
 			}
-			AVR_LOG(vcd->avr, LOG_WARNING,
-					"%s is an invalid IRQ format\n",
-					vcd->signal[i].name);
+			if(dup) free(dup);
```

Comment:
This is the closest I could get to the indentation on line 311 and others (e.g. 207):  two tabs under AVR_LOG, which aligns to the paren when ts=4, and aligns to the comma when ts=8.  Can you tell me what I'm missing?  Or is it that github formats with ts=8 and that looks wrong?